### PR TITLE
Python numerical types

### PR DESCRIPTION
**Pull request by fm-andres-rangel**

### CODE REVIEW

File: python/py_rational64.hpp [hunk 1]
```diff
@@ -0,0 +1,569 @@
...
+PyObject *ExtractorBaseTypeRational64::nb_float(PyObject *self) {
+  double dest;
+  fmc_rational64_to_double(&dest, &((ExtractorBaseTypeRational64 *)self)->val);
+  return PyFloat_FromDouble(dest);
+}
+
+PyObject *ExtractorBaseTypeRational64::nb_int(PyObject *self) {
+  double dest;
+  fmc_rational64_to_double(&dest, &((ExtractorBaseTypeRational64 *)self)->val);
+  return PyLong_FromLong(llround(dest));
```
Comment (fm-ivang) [inline python/py_rational64.hpp:149]: FromLongLong

File: python/py_rprice.hpp [hunk 2]
```diff
@@ -0,0 +1,555 @@
...
+PyObject *ExtractorBaseTypeRprice::nb_float(PyObject *self) {
+  double dest;
+  fmc_rprice_to_double(&dest, &((ExtractorBaseTypeRprice *)self)->val);
+  return PyFloat_FromDouble(dest);
+}
+
+PyObject *ExtractorBaseTypeRprice::nb_int(PyObject *self) {
+  int64_t dest;
+  fmc_rprice_to_int(&dest, &((ExtractorBaseTypeRprice *)self)->val);
+  return PyLong_FromLong(dest);
```
Comment (fm-ivang) [inline python/py_rprice.hpp:144]: FromLongLong

File: test/numerical.py [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+    assert v1 == extr.Rational64(25)
+    v1 = extr.Rational64(10)
+    v1 /= extr.Rational64(2)
+    assert v1 == extr.Rational64(5)
+
+    assert max(extr.Rational64(10), extr.Rational64(2), extr.Rational64(5)) == extr.Rational64(10)
+    assert min(extr.Rational64(10), extr.Rational64(2), extr.Rational64(5)) == extr.Rational64(2)
+
+    v1 = extr.Rational64.from_float(2.35)
+    assert float(v1) == 2.34375
```
Comment (fm-ivang) [inline test/numerical.py:304]: Add test for overflow

**Review by fm-ivang — CHANGES_REQUESTED** [review]

**Review by fm-ivang — APPROVED** [review]

**Review by maxim-trokhimtchouk — APPROVED** [review]

**Review by maxim-trokhimtchouk — APPROVED** [review]
